@navikt/core/css/progress-bar.css

```diff
@@ -0,0 +1,95 @@
+.navds-progress-bar {
+  background: var(--ac-progress-bar-bg, var(--a-surface-neutral-subtle));
```

Comment:
Kan gjøre det enkelt og bare tilby token for foreground. Siden default bakgrunn en nøytral vil det ikke være nødvendig å overskrive den i de fleste tilfeller 🤔

Reply:
Mulig, men hvis man først vil overskrive den ene så har man kanskje en idé om den andre også?

Reply:
Tanken var bare siden bakgrunnsfargen ikke er koblet til noen branding eller variant (success, warning, error etc), så vil være lite brukt.